### PR TITLE
Added 03_finetune_distilBERT

### PR DESCRIPTION
**Pull request by JephthahBondzie**
notebook contains the codes for the distilBERT pretrained model
Fine-tuned a DistilBERT model for customer support classification

Key changes:

Added custom preprocessing ( tokenization )

Implemented training with hyperparameter tuning (epoch, learning rate, batch size).

Saved model and tokenizer for future inference.

### CODE REVIEW

**Comment by JephthahBondzie** [comment]
Please take time to review
